pyproject.toml

```diff
@@ -93,7 +93,7 @@ pytz = ">=2019.1,<2022.0"
 rasa-sdk = "^2.8.0"
 colorclass = "~2.2"
 terminaltables = "~3.1.0"
-sanic = ">=19.12.2,<21.0.0"
+sanic = "20.3.0"
```

Comment:
could we keep the range though?

Reply:
thing was just when I used `poetry update` it changed to 20.13.0 or so and then we had a ton of mypy errors - I think there was some problem with the stubs. I could have a look to see if I can actually resolve that

Reply:
so maybe `>=19.12.2,<20.13.0`? and maybe open a follow-up issue to upgrade sanic and do this investigation there?

Reply:
Had to use `>=19.12.2,<20.12.0` to not get mypy issues